src/views/affiliations.ts

```diff
@@ -54,10 +63,39 @@ export class AffiliationsView extends BlockView<Trackable<AffiliationNode>> {
     }
     this.version = affs.version
     this.container.innerHTML = ''
+
+    const comment = createCommentMarker('div', this.node.attrs.id)
```

Comment:
You can move event listener assignment into this function so you won't have to assign it in a different place and also you wont have to import 'handleCommentMarkerClick' separately. Additionally marker can be a class with this updateCommentSelection method on it. That way you can preserve marker's reference in 'this.marker' here and only call this.marker.updateCommentSelection(...) when need. Same for the handleComment function.
What do you think?